src/LookingGlassConfig.ts

```diff
@@ -245,11 +267,43 @@ export class LookingGlassConfig extends EventTarget {
 	}
 
 	/**
-	 * defines the quilt resolution, only change this at start, do not change this after an XRSession has started
+	 * defines the quilt resolution, if null, it will be set based on the connected device
 	 */
 
     public get quiltResolution(): number {
-		return this._viewControls.quiltResolution
+		if (this._viewControls.quiltResolution == null) {
+			const serial = this._calibration.serial
+			switch (true) {
+				case serial.startsWith("LKG-2K"): // Looking Glass 8.9"
+					return 4096
```

Comment:
Our default for this device was actually 2048, but this is not critical to change.